artifacts/definitions/Linux/Remediation/Quarantine.yaml

```diff
@@ -0,0 +1,179 @@
+name: Linux.Remediation.Quarantine
+description: |
+  This artifact applies quarantine to Linux systems via nftables.
+  It expects the target system to have nftables installed, and
+  hence the availability of nft CLI.
+
+  This artifact will create a table, with the default name
+  *vrr_quarantine_table*, which contains three chains. One
+  for inbound traffic, one for outbound traffic, and the other
+  for forwarding traffic. The chains will cut off all traffics
+  except those for DNS lookup and velociraptor itself.
+
+  To unquarantine the system, set the *RemovePolicy* parameter to *True*.
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: pathToNFT
+    default: /usr/sbin/nft
+    description: We depend on nft to manage the tables, chains, and rules.
+
+  - name: TableName
+    default: vrr_quarantine_table
+    description: Name of the quarantine table
+
+  - name: MessageBox
+    description: |
+        Optional message box notification to send to logged in users. 256
+        character limit.
+
+  - name: RemovePolicy
+    type: bool
+    description: Tickbox to remove policy.
+
+sources:
+  - query: |
+     // If a MessageBox configured truncate to 256 character limit
+     LET MessageBox <= parse_string_with_regex(
+               regex='^(?P<Message>.{0,255}).*',
+               string=MessageBox).Message
+     // TODO(gyee): do we need to use the wall command or some equivalent to send the message?
```

Comment:
Yes, given our focus on servers, this should be fine. A more general approach would need libnotify notifications but I'm happy to consider that out of scope.

Reply:
I'll use the `wall` command for now and add a TODO there to switch over to libnotify later on.